src/fgdpp/fgdpp.cpp

```diff
@@ -486,7 +517,7 @@ bool FGD::ParseFile() {
             continue;
         }
 
-        if (iequals(iter->string, "@include")) {
+        if (iequals(iter->string, R"(@include)")) {
```

Comment:
Why are all these strings using raw string notation? They don't have any characters that would warrant it